com.vysp3r.ProtonPlus.json

```diff
@@ -0,0 +1,24 @@
+{
+    "app-id" : "com.vysp3r.ProtonPlus",
```

Comment:
I see no mention of `vysp3r.com` anywhere and the website returns 404. Is this domain under the control of upstrean?

Reply:
I own the domain, but do not currently have any site up.